addons/point_of_sale/static/src/utils.js

```diff
@@ -117,6 +117,10 @@ export function loadImage(url, options = {}) {
  * @param {HTMLElement} el
  */
 export function loadAllImages(el) {
+    if (!el) {
+        return Promise.resolve();
+    }
+
```

Comment:
I think it's better if we can assure that `el` is not null or undefined.

Reply:
And btw, is this necessary for the runbot error fix?

Reply:
Yep
![image](https://github.com/user-attachments/assets/aaf84a55-5680-429d-8e4e-13a14e76525d)

Reply:
I don't really know why sometime it can be undefined